src/main/java/swift/ui/MainWindow.java

```diff
@@ -233,11 +239,11 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
```

Comment:
Use a switch case with the enum to make it neater
```suggestion
switch(result_display) {
  case Display.HELP: ....
  case Display.EXIT: ....
  .....
}
```